web/helpers/issue.helper.ts

```diff
@@ -184,3 +184,21 @@ export function getChangedIssuefields(formData: Partial<TIssue>, dirtyFields: {
 
   return changedFields;
 }
+
+export const formatUserList = (users: string[]): string => {
```

Comment:
Minor change can you use generic names like formatTextList and take an argument of TextArray. So that we can reuse this for something like labels, modules etc... in property dropdowns

Reply:
yeah sure